app/src/main/java/fr/free/nrw/commons/explore/SearchActivity.java

```diff
@@ -31,12 +31,13 @@
 public class SearchActivity extends NavigationBaseActivity implements MediaDetailPagerFragment.MediaDetailProvider{
 
     @BindView(R.id.toolbar_search) Toolbar toolbar;
-    @BindView(R.id.searchBox) EditText etSearchKeyword;
+    @BindView(R.id.searchBox)
+    SearchView searchView;
 
     private SearchImageFragment searchImageFragment;
     private FragmentManager supportFragmentManager;
     private MediaDetailPagerFragment mediaDetails;
-    private String query;
+    private String str_query;
```

Comment:
Java variable should be using caseLikeThis rather than case_like_this

Reply:
I would suggest `queryString` or just `query` because we already know it is a String.